src/main/java/frc/trigon/robot/subsystems/roller/RollerIO.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.subsystems.roller;
+
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.roller.toohardroller.ToohardRollerIO;
+import org.littletonrobotics.junction.AutoLog;
+
+public class RollerIO {
+
+    protected void updateInputs(RollerInputsAutoLogged inputs) {
+    }
+
+    protected void setAngleMotorPower(double power) {
+    }
+
+    protected void setCollectionMotorPower(double power) {
+    }
+
+    protected void stopAngleMotor() {
+    }
+
+    protected void stopCollectionMotor() {
+    }
+
+    @AutoLog
+    protected static class RollerInputs {
+        public double
+                angleMotorCurrent = 0,
+                collectionMotorCurrent = 0;
+        public double
+                angleMotorVoltage = 0,
+                collectionMotorVoltage = 0;
```

Comment:
Here, due to the design pattern, we would have the extra `public double` for each value, as well as a whitespace to separate motors